hack/generated/pkg/reflecthelpers/reflect_helpers_test.go

```diff
@@ -58,23 +58,38 @@ func createDummyResource() *batch.BatchAccount {
 	}
 }
 
+func MakeGroupToVersionMap(scheme *runtime.Scheme) (map[string]schema.GroupVersion, error) {
+	result := make(map[string]schema.GroupVersion)
+	for gvk := range scheme.AllKnownTypes() {
+		if gvk.Group == "" {
+			continue
+		}
+		result[gvk.Group] = schema.GroupVersion{Group: gvk.Group, Version: gvk.Version}
```

Comment:
Does this need to prevent overwriting a group if it's there twice?

Reply:
Since this is used purely in test it's not a big deal, but I've fixed this as part of the refactor to solve the group-kind issue mentioned above.